src/replicache.ts

```diff
@@ -356,6 +359,17 @@ export class Replicache<MD extends MutatorDefs = {}> {
     this.puller = puller;
     this.pusher = pusher;
 
+    const internalOptions = options as {
+      disableLicensing?: boolean;
+      disableMutationRecovery?: boolean;
+    };
+    if (internalOptions.disableLicensing) {
+      this._disableLicensing = true;
+    }
+    if (internalOptions.disableMutationRecovery) {
+      this._disableMutationRecovery = true;
+    }
+
```

Comment:
I would do something more along the lines of:

```ts
type InternalOptions = {
  disableLicensing?: boolean;
  disableMutationRecovery?: boolean;
};
const {
  disableLicensing = false,
  disableMutationRecovery = false,
} = options as InternalOptions;
this._disableLicensing = disableLicensing;
this._disableMutationRecovery = disableMutationRecovery;
```

And then remove the initializer in the class field declarations.